### DISCUSSION TITLE
专业版突然变成免费版，请协助恢复

### DISCUSSION THREAD

**Discussion by casperwzy** [getlantern/lantern-forum-cn#276] [General]
3月26日，不知道为什么专业版就突然变成免费版了，今天依旧是免费版，相关的付款凭证，和账号邮箱已经发送电子邮件至 [support@getlantern.org]，请尽快帮助恢复，感谢！

**Comment by MaxMKMax** [comment]
@casperwzy 嗯嗯. 明白您的问题了. 没事, 我来帮您问一下. 请您提供我您购买使用的邮件.

**Reply by casperwzy** [reply]
[330522261@qq.com](mailto:330522261@qq.com)      

![Uploading webwxgetmsgimg_&MsgID=3641357987684934163&skey=%40crypt_a1c21476_ed67aa3c57a0a5af1d1ae784af247af7.jpg…]()

**Comment by casperwzy** [comment]
330522261@qq.com

**Reply by MaxMKMax** [reply]
@casperwzy 好的. 帮您提交问题上去了. 您等一等. 团队在开会, 今晚深夜会帮您弄. 您也留意邮箱.

**Reply by casperwzy** [reply]
好的，多谢，我会留意的

**Reply by MaxMKMax** [reply]
@casperwzy 亲, 不客气呢. 如果两天后还没弄好您再回来找我. 我看是否能帮您紧急处理.

**Reply by casperwzy** [reply]
好的，今天还是免费版，我继续等等

**Reply by casperwzy** [reply]
对了，在等待期间，想问一个问题，看你有说蓝灯可以重新登录，为什么我自从很多年前开始使用时登录过一次，然后一直都没登录再次登录过，现场怎么也找不到在哪里能登录，一直都是显示升级到专业版，请问，在哪里我可以试试重新登录呢？
![360截图20240328125814486](https://github.com/getlantern/lantern-forum-cn/assets/25656932/0018ae83-513a-4d42-b45f-c4ebf8a9a44f)

**Reply by MaxMKMax** [reply]
@casperwzy 您好. 就是按升级到专业版那里登陆. 

另一个方法是卸载了, 重新下载重新登录.

**Comment by MaxMKMax** [comment]
@casperwzy 
对了, 我们的客服或负责人给您发邮件了. 您看一看邮件. 里面好像又重新登陆的激活码什么的.

**Comment by MaxMKMax** [comment]
<img width="269" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/c47074e5-0509-45f4-8579-e1f2aba20d8f">

@casperwzy 聊天记录. 他说给您发邮件了. 您找看邮箱或垃圾邮箱.

**Reply by casperwzy** [reply]
收到了，已经登上专业版了，感谢！
